obs.py

```diff
@@ -674,70 +850,69 @@ def reduce_image(self):
                     ) > datetime.timedelta(
                         minutes=self.config["solve_timer"]
                     ):
-
-                        try:
-                            solve = platesolve.platesolve(
-                                paths["red_path"] + paths["red_name01"], pixscale
-                            )  # 0.5478)
-                            print(
-                                "PW Solves: ",
-                                solve["ra_j2000_hours"],
-                                solve["dec_j2000_degrees"],
-                            )
-                            target_ra = g_dev["mnt"].current_icrs_ra
-                            target_dec = g_dev["mnt"].current_icrs_dec
-                            solved_ra = solve["ra_j2000_hours"]
-                            solved_dec = solve["dec_j2000_degrees"]
-                            solved_arcsecperpixel = solve["arcsec_per_pixel"]
-                            solved_rotangledegs = solve["rot_angle_degs"]
-                            err_ha = target_ra - solved_ra
-                            err_dec = target_dec - solved_dec
-                            solved_arcsecperpixel = solve["arcsec_per_pixel"]
-                            solved_rotangledegs = solve["rot_angle_degs"]
-                            print(
-                                " coordinate error in ra, dec:  (asec) ",
-                                round(err_ha * 15 * 3600, 2),
-                                round(err_dec * 3600, 2),
-                            )  # NB WER changed units 20221012
-
-                            # IS IMAGE PART OF A SMARTSTACK? 1=YES
-                            smartStack = 0
-                            if (
-                                smartStack != 1
-                            ):  # We do not want to reset solve timers during a smartStack
+                        if smartstackid == 'no':
```

Comment:
I'm a bit confused on what the acceptable values for `smartStack` and `smartstackid` are. Are these both expected to be either "yes" or "no", and `smartstackid` can also be a date string?

Reply:
SmartStack is deleted, thats something I used while testing. smartstackid is either a str(number) that is just a smoosh together of the current time. It can also be 'no'. Both of these values go into the fits header for later processing post-BANZAI. smartstackid is also used to name a temporary file that holds onto the smartstack as it is being built.